pkg/controller/proposal/controller.go

```diff
@@ -642,9 +643,10 @@ func (r *Reconciler) reconcileApply(ctx context.Context, proposal *configapi.Pro
 			changeValues = proposal.Status.RollbackValues
 		}
 
+		updChangeValues := doCascadingDelete(changeValues, config.Values)
```

Comment:
I think `upd` is a bit odd - suggest `updated`

Reply:
Changed per suggestion